src/main/java/com/synopsys/integration/detect/workflow/componentlocationanalysis/ComponentLocatorLibraryInput.java

```diff
@@ -0,0 +1,20 @@
+package com.synopsys.integration.detect.workflow.componentlocationanalysis;
+
+import java.util.List;
+
+/**
+ * This class adheres to Component Locator Library's input schema.
+ * Any changes made here to the expected input should be accompanied by changes in the library and vice versa.
+ */
+public class ComponentLocatorLibraryInput {
```

Comment:
Just Input as name is sufficient.

Reply:
Renamed to **ComponentLocatorInput** because the main package name stayed as "componentlocationanalysis" as per comment here: https://github.com/blackducksoftware/synopsys-detect/pull/789/files/66da279efcc6bb7e72847e3682724acc372a4c67#r1238890780. 

If we end up renaming the package to **componentlocator** then we can rename this to simply **Input**.